.github/actions/build-dev-builder-images/action.yml

```diff
@@ -63,7 +105,22 @@ runs:
           IMAGE_NAMESPACE=${{ inputs.dockerhub-image-namespace }} \
           IMAGE_TAG=${{ inputs.version }}
 
-    - name: Build and push dev-builder-android image # Only build image for amd64 platform.
+        docker push ${{ inputs.dockerhub-image-registry }}/${{ inputs.dockerhub-image-namespace }}/dev-builder-centos:${{ inputs.version }}
```

Comment:
It's no need to build the image again. You should use `skopeo` to copy the image and push to ECR.